src/shared/components/plots/BoxScatterPlot.tsx

```diff
@@ -182,16 +183,30 @@ export default class BoxScatterPlot<D extends IBaseBoxScatterPlotPoint> extends
         return this.chartWidth - 20;
     }
 
+    @computed get horizontalLegendHeight() {
+        //height of legend in case its horizontal
+        if (!this.props.legendData) {
+            return 0;
+        } else {
+            const numRows = Math.ceil(this.props.legendData.length / LEGEND_ITEMS_PER_ROW);
+            return 23.7*numRows;
+        }
+    }
+
     private get legend() {
         const x = this.legendX;
         if (this.props.legendData && this.props.legendData.length) {
+            // all the props conditional on this.props.horizontal are because
+            //  we want to put the legend on top of the plot, since the plot
+            //  could be arbitrarily wide with a lot of boxes
             return (
                 <VictoryLegend
-                    orientation="vertical"
+                    orientation={this.props.horizontal ? "vertical" : "horizontal"}
```

Comment:
so there is still a case where legend will be vertical?

Reply:
yes, if the box plot is oriented so that more boxes go down instead of to the right. what do you think?